drizzlepac/pars/hap_pars/mvm_parameters/acs_wfc_index.json

```diff
@@ -4,8 +4,6 @@
     },
     "astrodrizzle": {
         "acs_wfc_any_n2": "acs/wfc/acs_wfc_astrodrizzle_any_n2.json",
-        "acs_wfc_any_n4": "acs/wfc/acs_wfc_astrodrizzle_any_n4.json",
-        "acs_wfc_any_n6": "acs/wfc/acs_wfc_astrodrizzle_any_n6.json",
```

Comment:
Line 8 ("n6") should not be deleted and the associated file should not be deleted.